lib/tasks/heart_rate_data.rake

```diff
@@ -0,0 +1,8 @@
+namespace :heart_rate_data do
+  desc "Update heart rate data for each user for Fitbit"
+  task sync: :environment do
+    HeartRateData.all.each do |heart_rate_data|
+      heart_rate_data.update_heart_rate_data!
+    end
```

Comment:
Given we've learned a bit more about the fitbit API, I think this needs to be refactored. Can you re-write this to select the last user, refresh the API token, create a heart_rate_data record and then update it with the current date's data please?